Geoms/CMS-2017.cc

```diff
@@ -344,36 +344,42 @@ namespace
     SetupIterationParams(ii[1].m_params, 1);
     ii[1].set_iteration_index_and_track_algorithm(1, (int) TrackBase::TrackAlgorithm::highPtTripletStep);
     ii[1].set_seed_cleaning_params(2.0, 0.018, 0.018, 0.018, 0.018, 0.018, 0.05, 0.018, 0.05); 
+    ii[1].set_dupl_params(0.5, 0.03,0.05,0.08);
```

Comment:
(just to remind) These need to be propagated to the CMSSW setup (via the JSON dumps). Do I understand correctly that the new parameters (`drth_*`) would need to be added to the JSON "schema"?

Reply:
@makortel: this seems correct, similarly to PR #320.

Reply:
I can look into it and update the PR. let me know